egs/aishell2/s5/run.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Copyright 2018 AIShell-Foundation(Authors:Jiayu DU, Xingyu NA, Bengu WU, Hao ZHENG)
+#           2018 Beijing Shell Shell Tech. Co. Ltd. (Author: Hui BU)
+# Apache 2.0
+
+# AISHELL-2 provides:
+#  * a Mandarin speech corpus (~1000hrs), free for non-commercial research/education use
+#  * a baseline recipe setup for large scale Mandarin ASR system
+# For more details, read $KALDI_ROOT/egs/aishell2/README.txt
+
+# modify this to your AISHELL-2 training data path
+# e.g:
+# trn_set=/disk10/data/AISHELL-2/iOS/data
+# dev_set=/disk10/data/AISHELL-2/iOS/dev
+# tst_set=/disk10/data/AISHELL-2/iOS/test
+trn_set=
+dev_set=
+tst_set=
+
+nj=20
+stage=1
+gmm_stage=1
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+# we should probably move jieba(for word segmentation) into 
+# Kaldi's "tools" dir with Dan's approval
+# before you run the entire recipe, run the following command
+# local/install_jieba.sh local/jieba
```

Comment:
can you please move the install_jieba.sh script into the tools directory and make it so that this script dies with an informative error message (telling you exactly what to run) if it's not installed?

Reply:
OK. I'll make it into env.sh as other tools do.